vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -1,13 +1,24 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this file,
-// You can obtain one at http://mozilla.org/MPL/2.0/.
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
 module ledger.mojom;
 
+enum ContributionStep {
+  STEP_AC_TABLE_EMPTY = -4,
```

Comment:
Can I ask why start steps at `-4`?

Reply:
I have no idea- the original PR does this (this is just an uplift)
https://github.com/brave/brave-core/pull/4372

Maybe @tmancey can provide more context?